html_app/ui/microscopy.gss

```diff
@@ -1368,6 +1368,7 @@ top: 49px;
 	bottom: -17px;
 	overflow:hidden;
 	background:black;
+	cursor: pointer;
```

Comment:
That seems odd to me. Can you give me an example?

http://www.w3.org/TR/css3-ui/#cursor says `grab`:

> Indicates that something can be grabbed (dragged to be moved). Often rendered as the backside of an open hand.

If we're already using `grab` elsewhere, how about `move` ?